test/integration/tasks.test.js

```diff
@@ -1516,7 +1516,7 @@ describe("Tasks", function () {
       expect(response.body).to.be.deep.equal({
         message: "Discord details of users with status missed updates fetched successfully",
         data: {
-          usersToAddRole: [activeUserWithProgressUpdates.discordId],
+          usersToAddRole: [],
```

Comment:
Why it is empty?
and add test case related to maven part

Reply:
I have fixed the test and have reverted back to old code